src/node/build/buildPluginCss.ts

```diff
@@ -117,10 +118,14 @@ export const createBuildCssPlugin = ({
           })
         }
 
+        let namedExports =
+          modulesOptions?.localsConvention === 'camelCase' ||
+          modulesOptions?.localsConvention === 'camelCaseOnly'
+
```

Comment:
This will be better if we extract logic into `codegenModuleCss` function.

Reply:
Ahh, so that's why.

Reply:
Sorry, but what should the `codegenModuleCss` be?